### PR TITLE
Coverage whezz2

### PR DESCRIPTION
**Pull request by winsvega**
Slice of https://github.com/ethereum/cpp-ethereum/pull/2529 
without mining tests

### CODE REVIEW

File: test/TestHelper.cpp [hunk 1]
```diff
@@ -250,18 +250,18 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 		assert(_o.count("value") > 0);
 		assert(_o.count("data") > 0);
 
-		if (bigint(_o["nonce"].get_str()) >= c_max256plus1)
+		if (bigint(_o.at("nonce").get_str()) >= c_max256plus1)
```
Comment (CJentzsch) [inline test/TestHelper.cpp]: why do you use at?

**Comment by CJentzsch** [comment]
Wait until the block branch is merged and rebase on top of that. Why did you use .at all over the place?

**Comment by winsvega** [comment]
Gavin asked to make const& in importTransaction.
json_spirit::mObject const& _o

**Comment by gavofyork** [comment]
ok i must admit that was unintended :(

const-correctness vs code readability.

**Pull request merged**